cutils.c

```diff
@@ -41,7 +41,7 @@
 #pragma GCC visibility push(default)
 #endif
 
-void pstrcpy(char *buf, int buf_size, const char *str)
+void js__pstrcpy(char *buf, int buf_size, const char *str)
```

Comment:
Out of curiosity, why 2 underscores?

Reply:
I saw several functions generic sounding down there like `js__hrtime_ns` using two, so I went with that.

Reply:
It's a convention Ben and I carried on from libuv and started applying here too.

Double underscore for internal stuff.